webportal/src/app/user/user-view/user-view.component.scss

```diff
@@ -0,0 +1,78 @@
+.divider {
+  border-top: 1px solid #7c7c7c;
```

Comment:
What's the reason of customize so much css style? Can we reuse the existing css style as this page should keep the same style as other pages?